tsconfig.json

```diff
@@ -18,7 +18,8 @@
     "strictNullChecks": true,
     "noUnusedLocals": true,
     "allowSyntheticDefaultImports": true,
+    "types": ["jest"],
     "incremental": true
   },
-  "exclude": ["node_modules", "modules/**/dist/*", "modules/**/ts-tmp/*", "docs"]
+  "exclude": ["cypress", "node_modules", "modules/**/dist/*", "modules/**/ts-tmp/*", "docs"]
```

Comment:
Storybook + Webpack will look at the Cypress folder for Typescript errors, but that should be handled inside the `cypress/tsconfig.json` instead.